dashboard-api/seed_cockroachdb.sql

```diff
@@ -16,7 +16,7 @@ VALUES
 
 CREATE TABLE studies(
     id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
-    user_id string NOT NULL REFERENCES users(id), 
+    user_id string NOT NULL REFERENCES users(id) ON DELETE CASCADE, 
```

Comment:
I like how enforcing the schema allows you to _discover_ potential future problems early on.